st2mistral/actions/stackstorm.py

```diff
@@ -77,19 +50,10 @@ def _build_callback_url(action_context, version='v2'):
         return None
 
 
-def _retry_on_exceptions(exc):
-    LOG.warning('St2Action failed with exception type %s. Determining '
-                'if action execution can be retried...', type(exc))
-
-    is_connection_error = isinstance(exc, requests.exceptions.ConnectionError)
-
-    return is_connection_error
-
-
 class St2Action(base.Action):
 
     def __init__(self, action_context, ref, parameters=None, st2_context=None):
-        if not st2_context or not st2_context.get('endpoint'):
+        if not st2_context or not st2_context.get('api_url'):
```

Comment:
Nvm, I see you added it here - https://github.com/StackStorm/st2/pull/2847/files#diff-58c009d73cbaa1394b7a3828e29590f3R158